views/accessibility-statement.html

```diff
@@ -0,0 +1,85 @@
+      <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we
+        respond to your complaint,
+        <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS)</a>.</p>
+
+      <h2 class="govuk-heading-m">Technical information about this service’s accessibility</h2>
+      <p>Companies House is committed to making its website accessible in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+      <p>This service is fully compliant with the
+        <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a>
+        AA standard.</p>
+
+      <h2 class="govuk-heading-m">How we tested this service</h2>
+      <p>We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA using a variety of manual and automated testing tools.</p>
+      <p>This service was last (internally) audited on 22 April 2021.</p>
+      <p>We tested all the screens from start page to confirmation screen.</p>
+
+      <h2 class="govuk-heading-m">What we’re doing to improve accessibility</h2>
+
+      <p>To improve this service's accessibility, we:
+        <ul class="govuk-list govuk-list--bullet">
+          <li>work with real users who have accessibility issues to make sure that our services meet their needs</li>
+          <li>carry out internal tests for compliance with Web Content Accessibility Guidelines V2.1 level A and level AA</li>
+          <li>work with external agencies in major UK cities to perform usability research with users</li>
+          <li>recruit customers through
+            <a href="https://www.gov.uk/government/news/help-improve-companies-house">our user panel</a>
+            to take part in sessions at our usability lab in Cardiff</li>
+          <li>build and test our services to meet
+            <a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements">government accessibility requirements</a>
+          </li>
+          <li>are developing
+            <a href="https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/">an assistive technology lab</a>
+            in our Cardiff office to improve the testing of our services and raise awareness of accessibility</li>
+
+          <p>
+            <a href="https://companieshouse.blog.gov.uk/category/user-research/">Read our blogs about usability</a>
+            to find out more about our user research work.</p>
+          <p>This statement was prepared on 22 April 2021. It was last updated on 22 April 2021.</p>
+
+        </div>
```

Comment:
the bottom 3 lines seem a bit out of alignment

Reply:
Amended